### ISSUE TITLE
Customer Value Co-Creation Platform

### ISSUE THREAD

**Issue by aditya-sambamoorthy** [district0x/district-proposals#282]
## **Name**: 
Customer Value Co-Creation Platform

## **Purpose**: 

The growing importance of technology and consumer feedback is shifting the epicenter of the value creation process from a product and firm centric view to a more consumer oriented view that revolves around an increased need for dialogue between the company, the end customers and the community creating personalised consumer-brand relationships. Emergence of new, penetrative and collaborative forms of interaction , facilitated by this age of social media, has empowered decentralised and dematerialised value exchange and consolidation like never before. The company can no longer view the crowd as a passive consumer of products but as a prospective collaborator in the value creation lifecycle. These trends conclusively demonstrate the need for customer value co-creation as the new age marketing paradigm to drive up consumer engagement levels, vouchsafing the relevance of the brand’s value proposition in a rapidly evolving socio-cultural context.

This platform will enable companies to collaborate with their consumers to design products, develop ideas and evolve branding strategies. The consumers can register on the platform to earn money by contributing their knowledge and opinions which will be evaluated through the principle of idea curation employed in curation markets. Popular suggestions will get upvotes and will earn money and so will sensible curation be incentivised.  This can help reduce production costs, decrease the time to market, increase the quality of innovation, encourage strong customer loyalty through superior consumer engagement and create new opportunities to monetise creativity.

## **Description**: 
Imagine this as a mobile app for end consumers and as a web interface for brands. Consumers can create their accounts and register their brand preferences on the app. Whenever their favourite brands announce co-creation challenges then the consumer gets notified. Based on their previous performance, the consumers are profiled by the platform for their skills for ex. the platform qualifies someone as a great apparel designer and someone else as an excellent branding strategist. These labels and levels not only encourage gamification but also introduce greater clarity in streamlining talent based on the situation/problem at hand. Excellent content creators and responsible content curators are rewarded on the platform and they're ranked through in-platform levels to measure attainment. 

## **Features**:
1. Brands can create/refine product/service design, ideation and branding challenges on the platform and tag it with a reward.
2. Users participate in the challenge based on their indicated interests and based on certain filter criteria if any as indicated by the brand while framing the challenge.
3. They can participate as content creators, content curators or both.
4. The challenge is either time bound or objective bound and once its met; the challenge is suspended.
5. The co-creation can also be bidirectional wherein the user can bring up a project/issue which can be taken up by brands.
6. The co-creation goes through multiple phases of design, assessment, simulation/prototyping and feedback.
7. The platform will be entirely managed by the community of end users and brands and not by any centralised institution so as to encourage greater dialogue , improved innovation and better transparency

**Comment by Bradymck** [comment]
Hey, please excuse the late reply. I wanted to say thank you for such a thoughtful and thorough marketplace idea. So would this be like a decentralized 99designs but with a marketing component included? 

There are a couple similar to this. You may want to reach out and think about collaboration! 
Pictosis seems like the closest to your idea. I believe their platform is life right now too but is lacking in some features you describe. The creator @Xibot is great too. 

Pictosis: https://github.com/district0x/district-proposals/issues/66

**Comment by aditya-sambamoorthy** [comment]
Yes it will be closely linked to decentralised idea generation, open innovation and customer value co-creation. I see it here as a tool to also improve consumer brand relationships and customer loyalty. So it will be more along the lines of that rather than a collaborative design based platform like pictosis or a talent recruitment/scouting platform.
All the platforms that support open innovation currently don't look at the entire lifecycle of design starting from ideation up until implementation feedback. This platform should be a one stop solution to solve "wicked problems" of any domain. Using the coordination framework provided by the incentive structure of blockchains , this platform can help various stakeholders work together using a range of collaborative tools to come up with solutions to complex and multi-pronged problems.

**Comment by Bradymck** [comment]
I was sending out rewards, can you post your ETH address please
